src/cmd/troubleshoot/logger.go

```diff
@@ -0,0 +1,44 @@
+package troubleshoot
```

Comment:
Is an implementation of a logger really needed? can't we reuse an existing implementation? e.g. `DTLogger` or an already existing library?

Reply:
It's needed. We want the output to be more human-readable.